torsions/TorsionScanSet.py

```diff
@@ -203,7 +203,7 @@ def compute_energy(self, param, platform=None):
             energy = state.getPotentialEnergy()._value
             mm_energy.append(energy)
             #print "frame %5d / %5d : %8.3f kJ/mol" % (i, self.n_frames, energy) # DEBUG
-            delta_energy.append(self.qm_energy[i]._value - energy)
+            delta_energy.append(self.qm_energy[i]._value - energy + offset)
```

Comment:
Not certain about the sign of `offset` here.  May need to be
```
delta_energy.append(self.qm_energy[i]._value - energy - offset)
```
Depends on the allowed limits of `offset`.  I'm not sure those are right either, since in principle we want something like
```
(QM-QM_zero) - (MM-MM_zero)
```
where both `QM_zero` and `MM_zero` are unknown, but likely more negative than the `min()` of any of their corresponding energies.